rules/falco_rules.yaml

```diff
@@ -24,7 +24,7 @@
 # change to this rules file, we'll uncomment this line and set it to
 # the falco engine version in use at the time.
 #
-#- required_engine_version: 2
+- required_engine_version: 3
```

Comment:
```suggestion
- required_engine_version: 4
```

Reply:
Or just rebase on top of the `dev` branch.

Reply:
Ok. Is 4 the minimum necessary for this check?

Reply:
?

Reply:
Yes

Reply:
ok done